DeAceroDLabs/marketplace-frontend
# feat: [DMP-672] Added port env variable.

## What is done

[[DMP-672] Added port env variable](https://deacero.atlassian.net/browse/DMP-672)

## How is done

- Before we needed to change manually every 3000 to 8200 when using docker.
- 8200 is when we are using the docker container on the front.
- 3000 is when we are running the back just locally.
- Added env variable `SERVICE_PORT` to Virtual Machine it is set at 3000 because localhost env variable on virtual machine is the container name:
![image](https://user-images.githubusercontent.com/81311646/198689670-0f0354ed-a2ea-46ec-9b2d-8b5e15cae358.png)


## How to test

1. To use container back run `npm run docker:run` on the back.
2. Add to the frontend env file this variable: `SERVICE_PORT=8200`
3. Run the project via `npm start`
4. Everything should work fine.
5. If you do not want to run back with docker, set variable to SERVICE_PORT to 3000.
